ydb/docs/en/core/reference/ydb-sdk/feature-parity.md

```diff
@@ -1,147 +1,132 @@
+| Retryer on the session pool (a repeat object is a session) | + | + | + | + | + | + | + | [![issue 216 status](https://github.ydb.tech/issues/ydb-php-sdk/216)](https://github\.com/ydb-platform/ydb-php-sdk/issues/216) |
+| Retryer on the session pool (a repeat object is a transaction within a session) | [![issue 455 status](https://github.ydb.tech/issues/ydb-cpp-sdk/455)](https://github\.com/ydb-platform/ydb-cpp-sdk/issues/455) | [![issue 595 status](https://github.ydb.tech/issues/ydb-python-sdk/595)](https://github\.com/ydb-platform/ydb-python-sdk/issues/595) | + | [![issue 400 status](https://github.ydb.tech/issues/ydb-java-sdk/400)](https://github\.com/ydb-platform/ydb-java-sdk/issues/400) | [![issue 458 status](https://github.ydb.tech/issues/ydb-js-sdk/458)](https://github\.com/ydb-platform/ydb-js-sdk/issues/458) | [![issue 324 status](https://github.ydb.tech/issues/ydb-dotnet-sdk/324)](https://github\.com/ydb-platform/ydb-dotnet-sdk/issues/324) | + | [![issue 217 status](https://github.ydb.tech/issues/ydb-php-sdk/217)](https://github\.com/ydb-platform/ydb-php-sdk/issues/217) |
+| Graceful session shutdown support | + | + | + | + | [![issue 459 status](https://github.ydb.tech/issues/ydb-js-sdk/459)](https://github\.com/ydb-platform/ydb-js-sdk/issues/459) | [![issue 325 status](https://github.ydb.tech/issues/ydb-dotnet-sdk/325)](https://github\.com/ydb-platform/ydb-dotnet-sdk/issues/325) | [![issue 274 status](https://github.ydb.tech/issues/ydb-rs-sdk/274)](https://github\.com/ydb-platform/ydb-rs-sdk/issues/274) | [![issue 218 status](https://github.ydb.tech/issues/ydb-php-sdk/218)](https://github\.com/ydb-platform/ydb-php-sdk/issues/218) |
+| Support for server-side load balancing of sessions | + | + | + | [![issue 401 status](https://github.ydb.tech/issues/ydb-java-sdk/401)](https://github\.com/ydb-platform/ydb-java-sdk/issues/401) | [![issue 460 status](https://github.ydb.tech/issues/ydb-js-sdk/460)](https://github\.com/ydb-platform/ydb-js-sdk/issues/460) | [![issue 326 status](https://github.ydb.tech/issues/ydb-dotnet-sdk/326)](https://github\.com/ydb-platform/ydb-dotnet-sdk/issues/326) | [![issue 275 status](https://github.ydb.tech/issues/ydb-rs-sdk/275)](https://github\.com/ydb-platform/ydb-rs-sdk/issues/275) | [![issue 219 status](https://github.ydb.tech/issues/ydb-php-sdk/219)](https://github\.com/ydb-platform/ydb-php-sdk/issues/219) |
```

Comment:
Support for server-side load balancing of sessions (a CreateSession request must contain the "session-balancer" value in the "x-ydb-client-capabilities" metadata header)